lib/rust/ensogl/component/text/src/component/area.rs

```diff
@@ -542,44 +542,27 @@ impl Area {
     //        pass information about scene layers they are assigned to. However, the [`GlyphSystem`]
     //        is a very non-standard implementation, and thus has to handle the new display object
     //        callbacks in a special way as well.
-    //     2. The `self.data.camera` has to still be used, otherwise there would be no way to
-    // convert        the screen to object space (see the [`to_object_space`] function). This is
-    // a very        temporary solution, as any object can be assigned to more than one scene
-    // layer, and thus        can be rendered from more than one camera. Screen / object space
-    // location of events        should thus become much more primitive information /
-    // mechanisms.     Please note, that this function handles the selection management
-    // correctly, as it uses     the new shape system definition, and thus, inherits the scene
-    // layer settings from this     display object.
+    //     2. The `self.data.layer.camera()` has to still be used, otherwise there would be no way
+    //        to convert the screen to object space (see the [`to_object_space`] function). This is
+    //        a very temporary solution, as any object can be assigned to more than one scene
+    //        layer, and thus can be rendered from more than one camera. Screen / object space
+    //        location of events should thus become much more primitive information / mechanisms.
+    //        Please note, that this function handles the selection management correctly, as it
+    //        uses the new shape system definition, and thus, inherits the scene layer settings
+    //        from this display object.
+    //     3. The `self.data.layer` needs to be stored so that the [`set_font`] function can assign
+    //        newly created Glyphs to the layer to make them visible.
```

Comment:
There is a lot of information in point 2 which is applicable also to point 3. Please merge these points, and make layer being a main topic (so explanations like `and thus can be rendered from more than one camera` are no longer required, because the camera is not a main topic, we don't store it).

Reply:
I edited it trying to merge the two points. If the result is not correct or could be improved in some other way, I'd be very grateful for a suggestion for the new text of the comment. Some of the aspects of the original comment are not fully clear to me, possibly due to not knowing some of the areas of the codebase well enough yet.